linchpin/provision/roles/libvirt/templates/libvirt_node.xml.j2

```diff
@@ -19,17 +19,17 @@
   <on_reboot>restart</on_reboot>
   <on_crash>destroy</on_crash>
   <devices>
-{% if definition[0]['driver'] is defined and definition[0]['driver'] == 'qemu' %}
+{% if definition[3]['driver'] is defined and definition[1]['driver'] == 'qemu' %}
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
 {% else %}
     <emulator>{{ qemu_path }}qemu-kvm</emulator>
 {% endif %}
     <disk type='file' device='disk'>
       <driver name='qemu' type='qcow2'/>
-      <source file='{{ definition[2] }}{{ libvirt_resource_name }}_{{ definition[1] }}.{{ definition[3] }}'/>
+      <source file='{{ definition[4] }}/{{ libvirt_resource_name }}_{{ definition[3] }}.{{ definition[5] }}'/>
```

Comment:
In the virt: define action above, the code moved `libvirt_resource_name` into the loop. It seems libvirt_resource_name is appropriate here, but it should be consistent.